test/LRPTests/APITests.cpp

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include <TestDef.h>
+#include <TlHelp32.h>
+#include <NotificationsLongRunningProcess_h.h>
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::System;
+
+namespace Test::LRP
+{
+    static const PCWSTR c_processName = L"TAEF.exe";
+    static const PCWSTR c_appId = L"toastAppId";
+
+    class LRPTests
+    {
+
+    public:
+        BEGIN_TEST_CLASS(LRPTests)
+            TEST_CLASS_PROPERTY(L"Description", L"Windows App SDK Push Notifications Long Running Process tests")
+            TEST_CLASS_PROPERTY(L"ThreadingModel", L"MTA")
+            TEST_CLASS_PROPERTY(L"RunAs:Class", L"RestrictedUser")
```

Comment:
What does this mean? Sounds like this is a non-admin user? Then how is it different from not specifying this property at all?

Reply:
RestrictedUser means that the process is running as a user at medium-IL. This is our only supported scenario, **but in the future** we will need to add ElevatedUser to make sure elevated processes can call into the LRP.

Reply:
What is the default IL we get for the tests if we don't specify a user?

Reply:
If you don't specify a user, TAEF will run in the context of which TAEF is running.